contracts/cw1-subkeys/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cw1-subkeys"
-version = "0.3.2"
+version = "0.4.0"
```

Comment:
Please don't update the version yet. There is a script to update all versions and we can do that once before publishing. (They all went from 0.11 -> 0.12, so everything is breaking)